assets/src/web-stories-block/block/components/storiesInspectorControls.js

```diff
@@ -0,0 +1,254 @@
+import { isShowing } from '../util';
+import AuthorSelection from './authorSelection';
+
+/**
+ * StoriesInspectorControls props.
+ *
+ * @typedef StoriesInspectorControlsProps
+ *
+ * @property {string} viewType String indicator of active view type.
+ * @property {number} numOfStories Number indicator of maximum number of stories to show.
+ * @property {number} numOfColumns Number indicator of number of columns in grid view type.
+ * @property {string} orderByValue String indicator of stories sorting.
+ * @property {boolean} isShowingTitle Whether or not to display story's title.
+ * @property {boolean} isShowingDate Whether or not to display story's date.
+ * @property {boolean} isShowingAuthor Whether or not to display story's author.
+ * @property {boolean} isShowingViewAll Whether or not to display stories archive link.
+ * @property {string} viewAllLinkLabel Archive link's label.
+ * @property {boolean} imageOnRight Whether or not to display images on right side in list view type.
+ * @property {Array} authors An array of authors objects which are currently selected.
+ * @property {Function} setAttributes Callable function for saving attribute values.
+ */
+
+/**
+ * LatestStoriesBlockControls component. Used for rendering block controls of the block.
+ *
+ * @param {StoriesInspectorControlsProps} props Component props.
+ *
+ * @return {*} JSX markup.
+ */
+const StoriesInspectorControls = (props) => {
+  const {
+    attributes: {
+      viewType,
+      numOfStories,
+      numOfColumns,
+      orderByValue,
+      viewAllLinkLabel,
+      authors,
+      sizeOfCircles,
+      fieldState,
+    },
+    setAttributes,
+    showFilters = true,
+  } = props;
+
+  const { fieldStates } = useConfig();
+
+  useEffect(() => {
+    // Set default field state on load.
+    if (!Object.entries(fieldState).length) {
+      setAttributes({
+        fieldState: fieldStates,
+      });
+    }
+  }, [fieldState, fieldStates, setAttributes]);
+
+  // Set up sort options.
+  const orderByOptions = Object.entries(ORDER_BY_OPTIONS).map(
+    ([key, option]) => {
+      return {
+        label: option.label,
+        value: key,
+      };
+    }
+  );
+
+  const previewLink = select('core/editor').getEditedPostPreviewLink();
+  const carouselMessage = sprintf(
+    /* Translators: Carousel informational message. 1: Preview link. */
+    __(
+      `<i><b>Note:</b> Carousel view's functionality will not work in Editor. <a target="__blank" href="%1$s">Preview</a> post to see it in action.</i>`,
+      'web-stories'
+    ),
+    previewLink
+  );
+
+  return (
+    <InspectorControls>
+      <PanelBody
+        className="web-stories-settings"
+        title={__('Story settings', 'web-stories')}
+      >
+        {CAROUSEL_VIEW_TYPE === viewType && (
+          <Notice
+            className="web-stories-carousel-message"
+            isDismissible={false}
+            status="warning"
+          >
+            <RawHTML>{carouselMessage}</RawHTML>
```

Comment:
Why does this need to use rawhtml?

Reply:
Also flagged here: https://github.com/rtCamp/web-stories-wp/pull/188#discussion_r555001186